lab/bom.md

```diff
@@ -0,0 +1,164 @@
+| AMD/Pensando DSC200              | AMD          |                    | 2        | $4, 000   | $4, 000  | $8, 000   | [Product](https://www.amd.com/system/files/documents/pensando-dsc-200-product-brief.pdf)                                                                                                                                                                               |
+| need full list of supported _PUs |              |                    |          |          | $0      | $0       |                                                                                                                                                                                                                                                             |
+|                                  |              |                    |          |          | $0      | $0       |                                                                                                                                                                                                                                                             |
+| Servers                          |              |                    |          |          | $0      | $0       |                                                                                                                                                                                                                                                             |
+| VM Hypervisor                    | Dell      | RAX QH12-22E4-2GPU | 1        |          | $26, 020 | $26, 020  | [Shop](https://www.thinkmate.com/quotation-request?a=YToxOntzOjI6ImlkIjtpOjYxOTQ2Njt9)                                                                                                                                                                              |
+| DPU Host (1 or 2 _PUs)           | Supermicro   | RAX XS4-11S3-10G   | 10       |          | $5, 131  | $51, 310  | [Shop](https://www.thinkmate.com)                                                                                                                                                                                                                                   |                                                                                                                                                                                                                                                         |
+| Storage Host                     | ASUS         | ESC8000A-E11       | 1        |          | $14, 000 | $14, 000  | [Product](https://www.asus.com/commercial-servers-workstations/esc8000a-e11/)                                                                                                                                                                                          |
```

Comment:
I think we can skip storage host, just use software defined storage on regular server

Reply:
removed from phase1